modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-3644[OSSM-3644] Previously, the federation egress-gateway received the wrong update of network gateway endpoints, causing extra endpoint entries. Now, the federation-egress gateway has been updated on the server side so it receives the correct network gateway endpoints.
+
+* https://issues.redhat.com/browse/OSSM-3595[OSSM-3595] Previously, `istio-cni` sometimes failed on {op-system-base} because SELinux did not allow `iptables-restore` to open files in `/tmp` directory. Now, SELinux passes `iptables-restore` via `stdin` instead of via a file.
```

Comment:
```suggestion
* https://issues.redhat.com/browse/OSSM-3595[OSSM-3595] Previously, `istio-cni` sometimes failed on {op-system-base} because SELinux did not allow `iptables-restore` to open files in the `/tmp` directory. Now, SELinux passes `iptables-restore` via `stdin` instead of via a file.
```
- Sorry, I missed it before, my bad. But all of the backticked literals in this sentence need referents. E.g.  what is `istio-cni`? a tool? a utility? an object? a component?

Reply:
@gwynnemonahan Please mark conversations like this one as resolved. Thanks.